code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm

```diff
@@ -281,6 +293,7 @@ obj/structure/elite_tumor/proc/onEliteWon()
 		SEND_SOUND(mychild, sound('sound/effects/magic.ogg'))
```

Comment:
I didn't catch this in the original pr but use `mychild.playsound_local` instead

Reply:
Just here, or everywhere SEND_SOUND is used?

Reply:
Generally everywhere, send sound is just raw sound sending, playsound/_local is wrapper that should be used anywhere that handles deafness/distance etc.